DQMServices/Demo/test/test_dqmgui_upload.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/env bash
+# Simple test to use one of the files generated by TestDQMServicesDemo test
+# to upload it to the dev DQMGUI, making sure that the ROOT file format generated
+# is compatible. See cmssw issue #43590.
+# Requires curl and jq to be installed. Also requires either a Grid key and certificate,
+# or a proxy to be available.
+set -x
+set -e
+
+DEV_DQMGUI_URL="https://cmsweb.cern.ch/dqm/dev"
+# Create a unique fake "Era", so that different executions of this test
+# produce differently named files. This is required, because this test might pass 
+# if it's checking a file that was successfully uploaded during a previous
+# instance of the same test.
+UNIQUE_ERA_ID=$(date -u +%Y%m%d%M%S%N)
+OLD_FILENAME=DQM_V0001_R000000001__Harvesting__DQMTests__DQMIO.root
+NEW_FILENAME=$(echo $OLD_FILENAME | sed "s/DQMTests/DQMTests${UNIQUE_ERA_ID}/")
+# File size, to verify later that the file is fully uploaded
+original_file_size=$(stat --printf="%s" $OLD_FILENAME)
+
+# TODO: Is there any way to not hardcode the required test name, TestDQMServicesDemo?
+UNIT_TESTS_BASE_PATH=${LOCALTOP}/unit_tests/TestDQMServicesDemo
```

Comment:
@nothingface0 , looks like this is not used , so do we need this?

Reply:
Yes, I'm cleaning up, thanks